Stripe/STPCard.m

```diff
@@ -135,6 +148,35 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+- (NSString *)description {
```

Comment:
I just dug into this a little bit. Calling `NSLog(card)` will use the `description` and not `debugDescription`. I feel like that's a common use case that we want to support.